test/ThriftTest.thrift

```diff
@@ -115,14 +115,6 @@ struct CrazyNesting {
   4: binary binary_field
 }
 
-union SomeUnion {
```

Comment:
I, uhh...am not sure why this got removed. I'll re-add it. Suffice it to say, the code handles unions.